src/flock.rs

```diff
@@ -130,9 +129,10 @@ impl Filesystem {
     ///
     /// Handles errors where other processes are also attempting to concurrently
     /// create this directory.
-    pub fn create_dir(&self) -> Result<(), VetError> {
-        fs::create_dir_all(&self.root)
-            .wrap_err_with(|| format!("failed to create directory `{}`", self.display()))
+    pub fn create_dir(&self) -> Result<(), std::io::Error> {
+        fs::create_dir_all(&self.root)?;
+        Ok(())
+        //    .wrap_err_with(|| format!("failed to create directory `{}`", self.display()))
```

Comment:
This context wrapping was actually inherited from cargo because the errors in the stdlib don't include the directory path information. Probably not a huge deal to drop though.

Let's drop the `?;\nOk(())` and the commented out code though?